### ISSUE TITLE
OSX: starting Shoes from command line dups app

### ISSUE THREAD

**Issue by ccoupe** [Shoes3/shoes3#137]
When launching from command line using shoes-launch, two copies of the app are run. One is hidden behind the other.  Should probably note in the wiki/readme/manual that the launch command line on OSX is pretty obscure. Paths must be fully expanded - although `~/` does work.

Assume that Shoes.app lives in `~/build/mavericks`. The command to launch is 
```
~/build/mavericks/Shoes.app/Contents/MacOS/shoes-launch ~/Projects/shoes3.2/bug136s.rb
```
That gets you two running windows for bug136s.rb.

Related to issue #20

**Comment by ccoupe** [comment]
This turns out to be a unintended side effect of #127 and noted in #20. shoes.rb (OSX) doesn't expect to have to parse arguments like `--console myscript.rb` .  In modern OSX there is an --args flag to `open` which Shoes/Optionparser can handle so that Shoes/OSX **could be** command line friendly  like Linux and cshoes.exe for Windows).

**Comment by ccoupe** [comment]
This bug is just how OSX does things. I was using shoes-launch and not the shoes script.
Fixed in 3.2.24 by  using cshoes script for command line launching that does the right thing more often.
